Tests/UnitTests/Networking/Responses/Fixtures/ProductsEntitlements.json

```diff
@@ -0,0 +1,23 @@
+{
+    "products": [
```

Comment:
Should this be `products_entitlements_mapping` or something like that?

Reply:
I based this off of the suggested format from the doc.